src/duckstation-qt/translations/duckstation-qt_zh-cn.ts

```diff
@@ -3638,7 +3634,7 @@ The saves will not be recoverable.</source>
     <message>
         <location filename="../qtprogresscallback.cpp" line="10"/>
         <source>DuckStation</source>
-        <translation>DuckStation</translation>
+        <translation></translation>
```

Comment:
Same here

Reply:
i don't know programming, i want ask you if you source code here is "DuckStation",translation below is still reserved necessarily?

Reply:
Seems that if `<translation></translation>` is blank, it uses the original string.

So, this is fine :)

Reply:
OK, all my translation cut is exactly same meaning to original string.